packages/update-monitor/src/common/utils/getMembers.ts

```diff
@@ -0,0 +1,12 @@
+export function getMembers(granted: string[], revoked: string[]): string[] {
+  const members: string[] = [...granted]
+
+  revoked.forEach((revokedAddress) => {
+    const index = members.indexOf(revokedAddress)
+    if (index !== -1) {
+      members.splice(index, 1)
+    }
+  })
```

Comment:
I like this design for its simplicity, but unfortunately it fails for a very weird and uncommon edge case:

- Add Joe
- Remove Joe
- Remove Joe (again, because the smart contract just sets false and emits)
- Add Joe

This function should return `['Joe']` but will return `[]`

I think that block numbers should be taken into account

Reply:
I thought about it, there is also logic inside smart contract to prevent such case. Maybe the more robust method can be coded in the next PRs? because this one handles our case without a problem, WDYT

Reply:
@sz-piotr